observability/otel/otel-collector/otel-collector-exporters.adoc

```diff
@@ -290,6 +293,153 @@ include::snippets/technology-preview.adoc[]
+
+:FeatureName: The AWS CloudWatch Logs Exporter
+include::snippets/technology-preview.adoc[]
+
+.OpenTelemetry Collector custom resource with the enabled AWS CloudWatch Logs Exporter
+[source,yaml]
+----
+# ...
+  config:
+    exporters:
+      awscloudwatchlogs:
+        log_group_name: "<group_name_of_amazon_cloudwatch_logs>" # <1>
+        log_stream_name: "<log_stream_of_amazon_cloudwatch_logs>" # <2>
+        region: <aws_region_of_log_stream> # <3>
+        endpoint: <service_endpoint_of_amazon_cloudwatch_logs> # <4>
+        log_retention: <supported_value_in_days> # <5>
+# ...
+----
+<1> Required. If the log group does not exist yet, it will be created automatically.
+<2> Required. If the log stream does not exist yet, it will be created automatically.
+<3> Optional. If the AWS region is not already set in the default credential chain, you must specify it.
+<4> Optional. The Amazon CloudWatch Logs service endpoint to which the requests are forwarded. link:https://docs.aws.amazon.com/general/latest/gr/cwl_region.html[See the CloudWatch Logs endpoints] for a list.
+<5> Optional. With this parameter, you can set the log retention policy for new CloudWatch log groups. If this parameter is omitted or set to `0`, the logs never expire by default. Supported values for retention in days are `1`, `3`, `5`, `7`, `14`, `30`, `60`, `90`, `120`, `150`, `180`, `365`, `400`, `545`, `731`, `1827`, `2192`, `2557`, `2922`, `3288`, or `3653`.
+
+[role="_additional-resources"]
+.Additional resources
+* link:https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/WhatIsCloudWatchLogs.html[What is Amazon CloudWatch Logs?] (AWS CloudWatch Logs User Guide)
+* link:https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials[Specifying Credentials] (AWS SDK for Go Developer Guide)
+// * link:https://docs.aws.amazon.com/sdk-for-go/v2/developer-guide/configure-auth.html[Configure Authentication] (AWS SDK for Go v2 Developer Guide)
+* link:https://docs.aws.amazon.com/general/latest/gr/cwl_region.html[Service endpoints] (AWS General Reference)
+
+[id="aws-emf-exporter_{context}"]
+== AWS EMF Exporter
+
+The AWS EMF Exporter converts the following OpenTelemetry metrics datapoints to the AWS CloudWatch Embedded Metric Format (EMF):
+
+- `Int64DataPoints`
+- `DoubleDataPoints`
+- `SummaryDataPoints`
+
+The EMF metrics are then sent directly to the AWS CloudWatch Logs service by using the `PutLogEvents` API.
+
+One of the benefits of using this exporter is the possibility to view logs and metrics in the CloudWatch console at `+https://console.aws.amazon.com/cloudwatch/+`.
+
+:FeatureName: The AWS EMF Exporter
+include::snippets/technology-preview.adoc[]
+
+.OpenTelemetry Collector custom resource with the enabled AWS EMF Exporter
+[source,yaml]
+----
+# ...
+  config:
+    exporters:
+      awsemf:
```

Comment:
We should also document log_retention and namespace. https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/awsemfexporter#exporter-configuration

Reply:
@IshwarKanse, please see my addition on line [361](https://github.com/openshift/openshift-docs/pull/88479/files#diff-4182f1a314619ad34871720858dd9e37c4c0797d1eb66028dfa410022a2eaa3cR361). Is this okay?